### ISSUE TITLE
The radar is not visible an a player's back anymore

### ISSUE THREAD

**Issue by sweet235** [Unvanquished/Unvanquished#3139]
That is the issue.

**Comment by DolceTriade** [comment]
As slipher points out, probably a bug with iqm migration.

**Comment by DolceTriade** [comment]
Indeed. The code wants `tag_head`, but the model has `head_tag` and `tag_gear`. Apparently, we want `head_tag`.

Should we fix in code or rename the model?

**Comment by VReaperV** [comment]
Do all tags start with `tag_`? Then we should change the model, to keep it consistent.

**Comment by DolceTriade** [comment]
We could also pull this out of the code and put in a config file, but yeah probably we should just fix the model. I was just trying to avoid a model release.

**Comment by illwieckz** [comment]
Proposed model fix:

- https://github.com/UnvanquishedAssets/res-players_src.dpkdir/pull/23
